notebooks/MAST/TESS/interm_gi_query/interm_gi_query.ipynb

```diff
@@ -0,0 +1,254 @@
+    "- **dataproduct_type** to specify that we want timeseries data\n",
+    "- **sequence_number** to specify that we want sector 1\n",
+    "- **proopsal_id** to specify the GI program number.  \n",
+    "\n",
+    "Remember that more than one GI can propose for the same target so we need wild cards around the name of the program in our query."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsCount = Observations.query_criteria_count(obs_collection = \"TESS\",\\\n",
+    "                                       dataproduct_type = [\"timeseries\"],\\\n",
+    "                                           sequence_number = 1,\\\n",
+    "                                          proposal_id = \"*G011183*\")\n",
+    "print(\"Number of Observations: %i\" % obsCount)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsTable = Observations.query_criteria(obs_collection = \"TESS\",\\\n",
+    "                                       dataproduct_type = [\"timeseries\"],\\\n",
+    "                                           sequence_number = 1,\\\n",
+    "                                          proposal_id = \"*G011183*\")\n",
+    "obsTable[0:5]['obsid','proposal_id','obs_id']\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"product_list\"></a>\n",
+    "## Retrieve the list of Data Products\n",
+    "\n",
+    "Next we use astroquery to retrieve the list of data products that are associated with each observation.  \n",
+    "\n",
+    "We will only ask for the data products associated with the first two.  The [0:2] can be removed from the code below to get all the observations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataProducts = Observations.get_product_list(obsTable[0:2])\n",
+    "dataProducts.columns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataProducts['obsID', 'productFilename', 'description']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download Light curves\n",
+    "<a id=\"download\"></a>\n",
+    "We limit our list of data products to just those with the **description** set to \"Light curves\" because we just want this type of data file. We then download those products and print out the manifest to show the download location. Finally we use fits.info to show the structure of the fits file. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "want = dataProducts['description'] == \"Light curves\"\n",
+    "print(dataProducts[want])\n",
+    "manifest = Observations.download_products(dataProducts[want])"
```

Comment:
I would move this line to the next cell, so you have the print statements in the same cells as the commands they are printing the output from.